core/api/src/domain/accounts/index.types.d.ts

```diff
@@ -107,6 +107,8 @@ type Account = {
   readonly contacts: AccountContact[]
   kratosUserId: UserId
   displayCurrency: DisplayCurrency
+  threadId: ThreadId | undefined
```

Comment:
is this assistant thread id?

Reply:
yes

Reply:
do you think I should rename it?

Reply:
I don't see why this needs to be on account at all...

rather should be on its own entity `SupportChat.id` or something like that.